token-swap/program/src/curve/stable.rs

```diff
@@ -36,15 +36,19 @@ fn compute_a(amp: u64) -> Option<u64> {
 
 /// Returns self to the power of b
 fn checked_u8_power(a: &U256, b: u8) -> Option<U256> {
-    let mut result = *a;
-    for _ in 1..b {
+    let mut result = U256::from(1);
+    for _ in 1..=b {
```

Comment:
if `b = 0`, shouldn't it be better to do ?
```suggestion
    for _ in 0..b {
```

Reply:
yes, thanks, it looks nicer